File: practice/codekata/begginner/set1/vowel.c
#include <stdio.h>

int main(void) {
char ch;
scanf("%c",&ch);
int b=(int)ch;
if(ch=='a'||ch=='e'||ch=='i'||ch=='o'||ch=='u')
	printf("vowel");
else
{
	if((b>=65&&b<=90)||(b>=97&&b<=122))
		printf("alphabet");
	else
		printf("consonant");
}
return 0;
}

File: beginning/Vowel.java
package gu;

import java.util.*;
public class VowelConsonants {
   public static void main(String args[]){
       char ch;
       Scanner s=new Scanner(System.in);
       ch=s.next().charAt(0);
      
       if(ch == 'a' || ch == 'e' || ch == 'i' || ch == 'o' || ch == 'u' )
            System.out.println("Vowel");
        else
            System.out.println("Consonant");

   } 
}